src/devcenter/azext_devcenter/custom.py

```diff
@@ -370,6 +460,15 @@ def _build_arguments_schema(cls, *args, **kwargs):
         return args_schema
 
 
+class PlanMemberCreate(_PlanMemberCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.member_id._required = True
+        args_schema.member_type._required = True
```

Comment:
Why not fix the requirement in swagger?

Reply:
Created an ADO task - will fix in next update for our 2024-06-01-preview API.